Libraries/StyleSheet/processColor.js

```diff
@@ -25,6 +26,21 @@ function processColor(color?: ?(string | number)): ?number {
     return undefined;
   }
 
+  if (typeof int32Color === 'object') {
```

Comment:
Should this variable get renamed too?

Reply:
Done.   I originally made this change in our fork of react-native where I'm in the habit of making the fewest number of changes possible so that merges from upstream are easier to diff.

Reply:
Done